src/SqlPersistence/SqlPersistence.csproj

```diff
@@ -1,51 +1,44 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard2.0</TargetFrameworks>
     <AssemblyName>NServiceBus.Persistence.Sql</AssemblyName>
     <RootNamespace>NServiceBus.Persistence.Sql</RootNamespace>
-    <AssemblyTitle>NServiceBus.Persistence.Sql</AssemblyTitle>
-    <Product>NServiceBus.Persistence.Sql</Product>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>$(SolutionDir)NServiceBus.snk</AssemblyOriginatorKeyFile>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <OutputPath>..\..\binaries\</OutputPath>
-    <NoWarn>$(NoWarn);CS0419</NoWarn>
```

Comment:
CS0419 = Ambiguous reference in cref attribute

Reply:
No warnings showed up when I removed this, so I left it out.